### DISCUSSION TITLE
Does gmx_MMPBSA support non-natrual amino acids like 'SEP', 'TPO', 'PTR' and terminus patch (NT2)

### DISCUSSION THREAD

**Discussion by narubi2** [Valdes-Tresanco-MS/gmx_MMPBSA#490] [Q&A] [answered]
Hello,

Questions: Does gmx_MMPBSA (for charmm36m) support 'SEP', 'TPO' and 'PTR' residues and NT2 patch?

I am trying to calculate protein-protein interaction using gromacs and gmx_MMPBSA.
There is ptm 'PTR' and I thought it was important so I decided to use 'PTR' residues.
Since charmm36m ff has 'PTR' information, I used charmm36m ff.
I also added NT2 using 'pdb2gmx -ter option'.
The simulations ran successfully.

I then calculated the binding affinity using mmPBSA from gmx_MMPBSA.
gmx_MMPBSA didn't show any errors or warnings.
But I am wondering if it is necessary to add 'PTR' information in 'residuetype.dat' file of gmx_MMPBSA, and I don't know if I have to do anything else.
I compared before and after the addition and found that there was no difference in the binding affinity value.

I'd be very grateful if anyone could help to clarify this.
Thanks in advance.

**Comment by marioernestovaldes** [comment, marked as answer]
If you use the -cp topol.top flag, it should work with any topology you generated before, including the one with the phosphorylated residues

**Reply by narubi2** [reply]
Ah, I really appreciate you clarifying my question.

Thank you very much and have a nice day.
